packages/ui/src/components/utils/border.styled.ts

```diff
@@ -20,7 +20,8 @@ type BorderColor =
   | 'input'
   | 'warning'
   | 'white'
-  | 'list';
+  | 'list'
+  | string;
```

Comment:
Do not add string here, add exact types if required.

Reply:
removed